apps/member-profile/app/routes/_profile.companies_.$id.tsx

```diff
@@ -154,6 +155,10 @@ export default function CompanyPage() {
             {company.levelsFyiSlug && (
               <LevelsFyiLink slug={company.levelsFyiSlug} />
             )}
+
+            {company.leetcodeTagged && (
+              <LeetcodeTaggedLink slug={company.leetcodeTagged} />
```

Comment:
we also need to change the name in these two lines

Reply:
And done!